department-of-veterans-affairs/abd-vro
# mv api gateway

## What was the problem?

We're not using this service, and it is cluttering our repos, SecRel pipelines, and codebase.

Associated tickets or Slack threads:
- #2697 

## How does this fix it?[^1]

- Moves the code to another repo which is linked on the homepage of our wiki

## How to test this PR
- Removing the deployment from escalating LHDI environments


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
